mono/metadata/class.h

```diff
@@ -292,6 +292,12 @@ mono_method_can_access_field (MonoMethod *method, MonoClassField *field);
 MONO_API mono_bool
 mono_method_can_access_method (MonoMethod *method, MonoMethod *called);
 
+MONO_API mono_bool
+mono_class_is_nullable (MonoClass *klass);
+
+MONO_API MonoClass*
+mono_class_get_nullable_param (MonoClass *klass);
```

Comment:
This one should probably be split up into an `_internal` version and a `MONO_RT_EXTERNAL_ONLY` `mono_class_get_nullable_param` which does a `MONO_ENTER_GC_UNSAFE`/`MONO_EXIT_GC_UNSAFE` transition.

`mono_class_from_mono_type_internal` can block in the runtime or access managed objects.

Reply:
Shouldn't all new apis be split, even if the external one doesn't mode switch?
I.e. isn't the goal that mono itself never call an external api?